source/aws-bootstrap-kit/lib/aws-config-recorder.ts

```diff
@@ -70,7 +70,7 @@ export class ConfigRecorder extends cdk.Construct {
 
     new cfg.CfnDeliveryChannel(this, 'ConfigDeliveryChannel', {
       s3BucketName: configBucket.bucketName,
-      name: "ConfigDeliveryChannel"
+      name: `ConfigDeliveryChannel-${this.node.addr}`
```

Comment:
I understand that you're trying to avoid name conflict here.  (CDK doesn't prefix this by default?)

What is `this.node.addr` and why do we use it instead of `${Stack.of(this).account}-${Stack.of(this).region}`?

My concern is whether it's a private var as I can't find any doc on this.  It's fine if that's common practice in cdk community though.

Reply:
It's one way to have unique resource name : see https://github.com/aws/constructs/pull/314 but this.node.addr . But indeed here it's not really necessary since an account can have only one of those anyway. will remove